.github/workflows/auto-approve.yml

```diff
@@ -26,9 +26,9 @@ jobs:
             echo "::set-output name=approved::true";
             exit 0
           fi
-          minor_regex='^(build\(deps\): )?[B|b]ump .+ from ([[:digit:]]+)\.([[:digit:]]+)\.([[:digit:]]+) to ([[:digit:]]+)\.([[:digit:]]+)\.([[:digit:]]+)'
+          minor_regex='build\(deps(-dev)*\): bump .+ from (\d{1,2})\.(\d{1,2})\.(\d{1,2}) to (\d{1,2})\.(\d{1,2})\.(\d{1,2})'
```

Comment:
Please add ^ at the beginning of the regex

Reply:
added